system-test/fixtures/google-gax-packaging-test-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-gax-packaging-test-app",
+  "version": "0.0.1",
+  "description": "Test application for google-gax",
+  "main": "src/index.js",
+  "files": [
+    "src",
+    "protos"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [],
+  "scripts": {
+    "lint": "eslint src/ samples/ system-test/ test/",
+    "prettier": "prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+    "start": "node src/index.js",
+    "test": "mocha test/*.js"
+  },
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-prettier": "^3.0.1",
+    "gts": "^1.0.0-0",
+    "mocha": "^6.1.4",
+    "prettier": "^1.17.0",
+    "through2": "^3.0.1",
+    "typescript": "~3.3.0"
+  },
+  "dependencies": {
+    "@grpc/grpc-js": "^0.3.6",
+    "google-gax": "file:./google-gax.tgz",
+    "grpc": "^1.20.0",
+    "lodash.merge": "^4.6.1",
```

Comment:
Have we carefully looked at why we need `lodash.merge` here?  Are we doing something `Object.assign` can't?

Reply:
We don't need `lodash.merge`, micro-generators will emit `Object.assign`. https://github.com/googleapis/gapic-generator-typescript/blob/0080e041a9488a8976c94464e2cd75c25e9e1107/src/v1alpha3/echo_client.ts#L96 Again, this is just a generated code, which I will eventually replace with TypeScript generated code (when it's ready).